djangocms_pageadmin/admin.py

```diff
@@ -272,7 +267,7 @@ def duplicate_view(self, request, object_id):
             form=form,
             object_id=object_id,
             duplicate_url=reverse(
-                "admin:{}_{}_duplicate".format(*info), args=(obj.pk,)
+                "admin:{}_{}_duplicate_content".format(*info), args=(obj.pk,)
```

Comment:
I reckon that if you're expecting the duplicate link to appear on the page admin changelist and it's important that it does, you might want to test for that (as in check the expected link is in `response.content`)